pkg/ddl/backfilling_merge_sort.go

```diff
@@ -41,21 +40,23 @@ type mergeSortExecutor struct {
 	cloudStoreURI       string
 	mu                  sync.Mutex
 	subtaskSortedKVMeta *external.SortedKVMeta
+	workerCntLimit      int
```

Comment:
In local add index, the workerCnt can be adjusted dynamiclly.
Need to discuss whether we will support the feature.

Reply:
> In local add index, the workerCnt can be adjusted dynamically. Need to discuss whether we will support the feature.

ok for now

Reply:
> In local add index, the workerCnt can be adjusted dynamically.

is this an exposed feature or just previous code can support use newest workerCnt for later subtask?

Reply:
@tangenta cc

Reply:
as it's an exposed behavior, i will revert the part that use saved worker-count
https://docs.pingcap.com/tidb/stable/ddl-introduction#balance-the-physical-ddl-execution-speed-and-the-impact-on-application-load-through-system-variables